### PR TITLE
Little language and formatting cleanups.

### PR DESCRIPTION
**Pull request by timsifive**

### CODE REVIEW

File: debug_module.tex [hunk 1]
```diff
@@ -442,7 +437,7 @@ \section{System Bus Access} \label{systembusaccess}
 access to devices that a hart does not have access to.
 \end{commentary}
 
-\section{Quick Access}
+\section{Minimally Intrusive Debugging}
```
Comment (mwachs5) [inline debug_module.tex]: Why change this title? Is there nowehre else we refer to Quick Access?

File: xml/dm_registers.xml [hunk 2]
```diff
@@ -435,8 +434,7 @@
       hold identifier information which is not
       further specified in this document.
 
-      The Device Tree itself is described in the RISC-V Privileged
-      Specification.
+      The Device Tree itself is described in the Privileged Spec.
```
Comment (mwachs5) [inline xml/dm_registers.xml]: Is it? It may have been removed.

Comment (timsifive) [reply xml/dm_registers.xml]: It's mentioned, but not defined anywhere. Instead it talks about a more handwavy "configuration string" and specifically mentions that platforms can provide a "configuration string pointer." I'll rename this register to reflect that.

File: debug_module.tex (hunk 1 above)
Comment (mwachs5) [reply debug_module.tex]: makes sense

File: xml/dm_registers.xml (hunk 2 above)
Comment (mwachs5) [reply xml/dm_registers.xml]: I thought we changed this to Device tree FROM config string because they were going to get rid of Config string... weird.

Comment (timsifive) [reply xml/dm_registers.xml]: From the preface to Version 1.9.1:
> Change configuration string proposal to be use a search process that supports various formats including Device Tree String and flattened Device Tree.

I don't know if I went and looked at the language when I changed the debug spec to refer to Device Tree.

**Comment by mwachs5** [comment]
If we are renaming Device Tree -> Configuration String again, weren't there some registers that we renamed to "Device Tree Valid" or something like that?

**Comment by mwachs5** [comment]
I am referring to: https://github.com/riscv/riscv-debug-spec/blob/472587910ce4796558fbb03b4c3a7ea29505010f/xml/dm_registers.xml#L89

**Comment by mwachs5** [comment]
Can we back out the device tree -> cfg string from this PR ,and make it a seperate one? I think the rest of this is benign and that's the only thing blocking my approval on it

**Comment by mwachs5** [comment]
Oh, I see that you already changed those registers name

**Pull request merged**